src/plugins/saved_objects_management/public/management_section/objects_table/components/table.tsx

```diff
@@ -253,6 +254,20 @@ export class Table extends PureComponent<TableProps, TableState> {
           );
         },
       } as EuiTableFieldDataColumnType<SavedObjectWithMetadata<any>>,
+      {
+        field: `updated_at`,
+        name: i18n.translate('savedObjectsManagement.objectsTable.table.columnUpdatedAtName', {
+          defaultMessage: 'Updated at',
+        }),
+        dataType: 'date',
+        sortable: true,
+        description: i18n.translate(
+          'savedObjectsManagement.objectsTable.table.columnUpdatedAtDescription',
+          { defaultMessage: 'Updated at of the saved object' }
```

Comment:
Minor: can we fix the grammar here?

Reply:
We can use `Last updated` for the column name and then change the description to:
`Last update of the saved object`